src/api/interfaces/ApiResult.ts

```diff
@@ -0,0 +1,11 @@
+// standardized return format for api result
+interface ApiResult {
+	success: boolean;
+	message: string;
```

Comment:
Question:

I looked at the usage of this `message` and it is used to store the i18n reason of either an error response or a success response. Will it be better if we provide an i18n key and the front end can provide the appropriate translation text depending on the locale of the user?

Reply:
The reason is a brief description of the API call result. It's not intended for direct display to users of the website, but as information for the frontend developers. It's possible to provide localized messages but seems a little unnecessary since the team is well-versed in english 😅